kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/api/Configuration.java

```diff
@@ -9,9 +9,16 @@
 
 public interface Configuration {
 
-    String KUBERNETES_MASTER = "kubernetes.master";
-    String KUBERNETES_NAMESPACE = "kubernetes.namespace";
-    String KUBERNETES_DOMAIN = "kubernetes.domain";
+    //Deprecated Property Names: {
+        String KUBERNETES_MASTER = "kubernetes.master";
```

Comment:
in favor of `master.url`, which I propose as the common new property name between cube-kubernetes and cube-openshift.